src/shared/dropdown-list-v2/dropdown-list.tsx

```diff
@@ -348,10 +348,12 @@ export const DropdownList = <T, V>({
                         tabIndex={active ? 0 : -1}
                         $active={active}
                     >
-                        {renderListItemIcon(selected)}
                         {renderListItem
                             ? renderListItem(item, { selected })
-                            : renderDropdownLabel(item, selected)}
+                            : [
+                                  renderListItemIcon(selected),
+                                  renderDropdownLabel(item, selected),
+                              ]}
```

Comment:
React expects array children to have a `key` prop

can render this in a Fragment instead

```suggestion
                            : <>
                                  {renderListItemIcon(selected)}
                                  {renderDropdownLabel(item, selected)}
                              </>}
```

Reply:
Sure, I've updated the array to a fragment